If I don't mount mount the host /tmp dir into the Girder service, run_r2d fails with the below error but the tag is still generated based, but based on the error output.
```
[2022-10-18 15:24:37,739] INFO: [Repo2Docker] Looking for /wholetale/repo2docker_config in /
[2022-10-18 15:24:37,743] INFO: [Repo2Docker] Loaded config file: /wholetale/repo2docker_config.py
[2022-10-18 15:24:38,010] INFO: Picked Git content provider.
[2022-10-18 15:24:38,017] INFO: fatal: repository '//tmp/tmpo5yvdhtd' does not exist
[2022-10-18 15:24:38,017] INFO: Traceback (most recent call last):
[2022-10-18 15:24:38,017] INFO: File "/usr/lib/python3.9/site-packages/repo2docker/contentproviders/git.py", line 33, in fetch
[2022-10-18 15:24:38,017] INFO: for line in execute_cmd(cmd, capture=yield_output):
[2022-10-18 15:24:38,017] INFO: File "/usr/lib/python3.9/site-packages/repo2docker/utils.py", line 29, in execute_cmd
[2022-10-18 15:24:38,018] INFO: raise subprocess.CalledProcessError(ret, cmd)
[2022-10-18 15:24:38,018] INFO: subprocess.CalledProcessError: Command '['git', 'clone', '--depth', '1', '//tmp/tmpo5yvdhtd', '/tmp/repo2dockermqnkhay2']' returned non-zero exit status 128.
[2022-10-18 15:24:38,018] INFO: The above exception was the direct cause of the following exception:
[2022-10-18 15:24:38,019] INFO: Traceback (most recent call last):
[2022-10-18 15:24:38,019] INFO: File "/usr/bin/jupyter-repo2docker", line 8, in <module>
[2022-10-18 15:24:38,019] INFO: sys.exit(main())
[2022-10-18 15:24:38,019] INFO: File "/usr/lib/python3.9/site-packages/repo2docker/__main__.py", line 443, in main
[2022-10-18 15:24:38,019] INFO: r2d.start()
[2022-10-18 15:24:38,019] INFO: File "/usr/lib/python3.9/site-packages/repo2docker/app.py", line 851, in start
[2022-10-18 15:24:38,020] INFO: self.build()
[2022-10-18 15:24:38,020] INFO: File "/usr/lib/python3.9/site-packages/repo2docker/app.py", line 752, in build
[2022-10-18 15:24:38,020] INFO: self.fetch(self.repo, self.ref, checkout_path)
[2022-10-18 15:24:38,020] INFO: File "/usr/lib/python3.9/site-packages/repo2docker/app.py", line 481, in fetch
[2022-10-18 15:24:38,020] INFO: for log_line in picked_content_provider.fetch(
[2022-10-18 15:24:38,020] INFO: File "/usr/lib/python3.9/site-packages/repo2docker/contentproviders/git.py", line 41, in fetch
[2022-10-18 15:24:38,020] INFO: raise ContentProviderException(msg) from e
[2022-10-18 15:24:38,020] INFO: repo2docker.contentproviders.base.ContentProviderException: Failed to clone repository from //tmp/tmpo5yvdhtd.
```

Mounting tmp into the girder container solves this.

--- a/docker-stack.yml
+++ b/docker-stack.yml
@@ -51,6 +51,7 @@ services:
       - ./src/gwvolman:/gwvolman
       - ./src/girderfs:/girderfs
       - /var/run/docker.sock:/var/run/docker.sock
+      - /tmp:/tmp
     environment:
       - DOMAIN=local.wholetale.org
       - DASHBOARD_URL=https://dashboard.local.wholetale.org

